aten/src/ATen/cpu/vec256/vec256_float_neon.h

```diff
@@ -0,0 +1,653 @@
+#pragma once
+
+// DO NOT DEFINE STATIC DATA IN THIS HEADER!
+// See Note [Do not compile initializers with AVX]
+
+#include <ATen/cpu/vec256/intrinsics.h>
+#include <ATen/cpu/vec256/vec256_base.h>
+// Sleef offers vectorized versions of some transcedentals
+// such as sin, cos, tan etc..
+// However for now opting for STL, since we are not building
+// with Sleef for mobile yet.
+
+namespace at {
+namespace vec256 {
+// See Note [Acceptable use of anonymous namespace in header]
+namespace {
+
+// For 32 bit arm rely on neon for 64 bit advanced vector extension
```

Comment:
I don't understand this comment.  We're not supporting 32-bit at all right now, right?

Reply:
I have lost the context myself :). I will update the comment. Mainly I think I want to point out that for aarch32 we will do inline asm to implement all the corresponding functions.